src/components/Select/Select.styles.js

```diff
@@ -17,8 +34,8 @@ export const SelectWrapper = styled.div`
     content: ' ';
     display: block;
     position: absolute;
-    top: 5.2rem;
-    right: var(--medium-space);
+    top: 2.5rem;
```

Comment:
Need to change this to 5rem or try 5.2rem. Whichever looks best.

Reply:
All done as advides please have a look.

Reply:
Looking good @admtomas. Just a couple of minor things on the Select.styles.js